exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -62,6 +63,15 @@ impl CommandExtension for Run {
                 "ejb-log-config-path",
                 false,
             ),
+            Argument::new_named(
+                EJB_LIBRARY_PATH,
+                false,
+                "Path to the directory containing the java_bindings shared library that is passed \
+                to JVM via -Djava.library.path. Mostly for internal usage.",
+                None,
+                "ejb-java-library-path",
```

Comment:
I'd pick the longer name `ejb-override-java-library-path` — because it overrides the normal mechanism.